routing/router.go

```diff
@@ -742,7 +742,7 @@ func (r *ChannelRouter) syncGraphWithChain() error {
 // usually signals that a channel has been closed on-chain. We do this
 // periodically to keep a healthy, lively routing table.
 func (r *ChannelRouter) pruneZombieChans() error {
-	var chansToPrune []uint64
+	chansToPrune := make(map[uint64]bool)
```

Comment:
similar comment here, this can just be a set of `map[uint64]struct{}`

Reply:
Done.